### DISCUSSION TITLE
有没有手机端？

### DISCUSSION THREAD

**Discussion by DreamPWJ** [DouyinFE/semi-design#196] [Q&A] [answered]
### Which Component Need Enhancement 期望新增功能的组件
- Button

**Comment by DreamPWJ** [comment]
非常期待手机端UI库

**Comment by wuhw** [comment, marked as answer]
移动端和跨端是我们比较看重的一个方向。但目前还没进入到详细调研与评审阶段，后续有新的结果我们会及时同步

**Comment by DreamPWJ** [comment]
期待，最好直接支持微信小程序或Flutter，虽然有难度，但是还是期望探索一下 @wuhw

**Comment by ColinChen2** [comment]
跨端确实很复杂，更新也快。但是可以考虑先上个H5的。

**Comment by guoChopper** [comment]
looking forward to the mobile version
